tests/nfs/install.pm

```diff
@@ -54,6 +54,8 @@ sub install_testsuite {
 
         install_dependencies_pynfs;
         assert_script_run("git clone -q --depth 1 $url $rel && cd ./pynfs");
+        #workaround poo#176907, don't use xdrlib3
+        assert_script_run('git fetch --depth 1 origin dfb0b07eb5c0579d34f321d27a9d3434f75be38a~; git checkout dfb0b07eb5c0579d34f321d27a9d3434f75be38a~');
```

Comment:
nit: I would just change `;` with `&&`, but not important (if the first command fails the second fails as well).